spec/factories/ticket_purchases.rb

```diff
@@ -4,5 +4,11 @@
     conference
     ticket
     quantity 10
+    paid false
+    factory :paid_ticket_purchase do
+    	after(:build) do |ticket_purchase|
```

Comment:
I think we should create tickets in after_create method because there are free ticket purchases as well that does not have any payment and if we do that we don't have to create tickets here.